glimpse/agrep/Makefile.NeXT

```diff
@@ -1,131 +0,0 @@
-# Copyright (c) 1994 Sun Wu, Udi Manber, Burra Gopal.  All Rights Reserved.
-
-# You might have to change these depending on your machine configuration.
-# AR and RANLIB are the library-archive programs. On IRIX, RANLIB is not
-# required (define it to true) and AR is in /usr/ccs/bin/ar (on our machine!).
-AR	      = /bin/ar
-RANLIB	      = /bin/ranlib
-
-# Define HAVE_DIRENT_H to be 1 when you don't have <sys/dir.h> else define it to be 0 (in this case, one of the other 3 flags may need to be defined to be 1).
-HAVE_DIRENT_H = 0
-HAVE_SYS_DIR_H	= 0
-HAVE_SYS_NDIR_H	= 0
-HAVE_NDIR_H	= 0
-
-# Define UTIME to be 1 if you have the utime() routine on your system. Else define it to be 0.
-UTIME = 1
```

Comment:
primarily because i don't like editing copies of the same files for no gain:
the first of the four commits i made (725528) was to remove the duplicate
identical directories that are not used by the build, a good number
of which had the problematic old license texts.

diff -ubr told me that
glimpse/agrep duplicates ./agrep
glimpse/compress duplicates ./compress
...and so on for glimpse/dynfilters, glimpse/index, glimpse/test
and glimpse/libtemplate.